lib/src/data/data_management.dart

```diff
@@ -244,6 +251,14 @@ class PreferenceProvider extends ChangeNotifier {
     notifyListeners();
   }
 
+  set addToPinnedApps(String value) {
```

Comment:
did you just make a setter call addToPinnedApps? i expect this to be a method and to be able to only add, not remove.

Reply:
It's just used like a toggle, maybe I should change the name of the method

Reply:
make it a method, not a setter, and call it like togglePinnedApp